README.md

```diff
@@ -218,6 +218,23 @@ postcss([
 ]);
 ```
 
+Another option is to use the default function for scoping variable names. The default function combines the filepath, name and hashed css into a single string and uses it as a variable name.
+
+```js
+const defaultLocalizeFunction = (
+  name: string,
+  filePath: string,
+  css: string
+) => {
+  const hash = crypto
+    .createHash('md5')
+    .update(css)
+    .digest('hex')
+    .slice(0, 6);
+  return `${filePath || 'default'}-${name}-${hash}`;
+};
+```
+
```

Comment:
Can you add docs on what the configuration would look like?

Reply:
sure thing!